src/worker.js

```diff
@@ -2,31 +2,51 @@ const { workerData, parentPort } = require("worker_threads");
 const chalk = require("chalk");
 const fs = require("fs").promises;
 const { ESLint } = require("eslint");
-
 const eslint = new ESLint(workerData.options);
-
+const path = require("path");
 const formatterPromise = workerData.formatter
   ? eslint.loadFormatter(workerData.formatter)
   : undefined;
 
+function getPrettyPath(absolutePath) {
+  let relativePath = path.relative(process.cwd(), absolutePath);
+  let containingDir = path.dirname(relativePath);
+
+  if (containingDir && containingDir !== '.') {
+      relativePath = path.join(path.basename(containingDir), path.basename(relativePath));
+  }
```

Comment:
What is the goal of this?

Reply:
To make the information clearer and the terminal more readable and tidier by not printing the entire path from hard disk root to the linted file. So:
- If the file is at the project root, print the file name only.
- If the file is in a subpath from project root, only print `containingDirectory/fileName.ext`

This information is usually enough to locate the problem. Especially if you're smart and also using the ESLint extension in your editor.